tsconfig.json

```diff
@@ -19,7 +19,8 @@
       "@spartacus/core": ["projects/core/public_api"],
       "@spartacus/storefront": ["projects/storefrontlib/src/public_api"],
       "@spartacus/vendor": ["projects/vendor/public_api"],
-      "@spartacus/assets": ["projects/assets/src/public_api"]
+      "@spartacus/assets": ["projects/assets/src/public_api"],
+      "@spartacus/incubator": ["projects/incubator/src/public"]
```

Comment:
should be `public_api`.

Also, I've tried to create a module in the incubator project and try to reference it in the app module using `@spartacus/incubator`, but could not get this to work. Have you tried this yourself?